src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs

```diff
@@ -440,14 +440,27 @@ void UpdateIsRefreshing(bool isInitialValue = false)
 					});
 				}
 				else
-					_refresh.Refreshing = isRefreshing;
+					_refresh.Refreshing = isRefreshing; 
+				
+				// Allow to disable SwipeToRefresh layout AFTER refresh is done
+				UpdateIsSwipeToRefreshEnabled();
 			}
 		}
 
 		void UpdateIsSwipeToRefreshEnabled()
 		{
 			if (_refresh != null)
-				_refresh.Enabled = Element.IsPullToRefreshEnabled && (Element as IListViewController).RefreshAllowed;
+			{
+				var isEnabled = Element.IsPullToRefreshEnabled && (Element as IListViewController).RefreshAllowed;
+				_refresh.Post(() =>
+				{
+					// NOTE: only disable while NOT refreshing, otherwise Command bindings CanExecute behavior will effectively
```

Comment:
Avoid that disabling ListView RefreshAllowed cancels refresh indicator on Android
https://github.com/xamarin/Xamarin.Forms/pull/14816